src/AppBundle/Resources/config/services.yml

```diff
@@ -1,4 +1,13 @@
 services:
+    ezplatform.pre_content_view_listener:
+        class: AppBundle\EventListener\PreContentViewListener
+        arguments:
+            - blogLocationId: '%blog.location_id%'
+        tags:
+            - name: kernel.event_listener
+              event: ezpublish.pre_content_view
+              method: onPreContentView
```

Comment:
could we use the same styling convention as symfony? 
[see app.exception_listener yaml example](http://symfony.com/doc/current/event_dispatcher.html)